dev/build/windows/makecoq_mingw.sh

```diff
@@ -337,16 +348,39 @@ function build_prep {
   fi
 }
 
+# ------------------------------------------------------------------------------
+# Like build_prep, but gets the data from an entry in ci-basic-overlay.sh
+# This assumes the following definitions exist in ci-basic-overlay.sh
+# $1_CI_BRANCH
+# $1_CI_GITURL
```

Comment:
This doc is slightly confusing since this function does not call `ci-basic-overlay.sh` but just uses the environment variables that were defined in this file and sourced before-hand.

Reply:
True. ci-basic-overlay.sh is called by make_addons, so this function only works for addons. This should be documented.

I guess I will document this and also use build_prep_overlay wherever it applies. It will get tomorrow, though.